.gitignore

```diff
@@ -48,3 +48,4 @@ Gemfile.lock
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+.vscode/*
```

Comment:
why is this removed?

Reply:
it's not, I told it to ignore that dir if it exists